packages/react-search-ui/README.md

```diff
@@ -40,58 +89,160 @@ export default function App() {
     <SearchProvider config={config}>
       {() => (
         <div className="App">
-          <Layout header={<SearchBox />} bodyContent={<Results />} />
+          <Layout
+            header={<SearchBox />}
+            bodyContent={<Results titleField="title" urlField="nps_link" />}
+          />
         </div>
       )}
     </SearchProvider>
   );
 }
 ```
 
-### SearchProvider <a id="searchprovider"></a>
+<a id="searchuiconfig"></a>
+
+## Configuring Search UI
 
-The SearchProvider is the core engine behind Search UI. It provides [actions](../search-ui/README.md#actions)
-and [state](../search-ui/README.md#state) in a React [Context](https://reactjs.org/docs/context.html). If
-you are familiar with Redux, you should recognize this model.
+[back](#nav)
 
-`SearchProvider` is essentially a wrapper around our [`search-ui`](../search-ui).
+Search UI is configured via the `SearchProvider`.
+
+Example:
+
+```jsx
+const config = {
+  apiConnector: connector,
+  disjunctiveFacets: ["acres"],
+  disjunctiveFacetsAnalyticsTags: ["Ignore"],
+  search_fields: {
+    title: {},
+    description: {}
+  },
+  result_fields: {
+    title: {
+      snippet: {
+        size: 100,
+        fallback: true
+      }
+    },
+    nps_link: {
+      raw: {}
+    },
+    description: {
+      snippet: {
+        size: 100,
+        fallback: true
+      }
+    }
+  },
+  facets: {
+    states: { type: "value", size: 30 },
+    acres: {
+      type: "range",
+      ranges: [
+        { from: -1, name: "Any" },
+        { from: 0, to: 1000, name: "Small" },
+        { from: 1001, to: 100000, name: "Medium" },
+        { from: 100001, name: "Large" }
+      ]
+    }
+  }
+};
+
+return (
+  <SearchProvider config={config}>
+    {() => (
+      <div className="App">
+        <Layout
+          header={<SearchBox />}
+          bodyContent={<Results titleField="title" urlField="nps_link" />}
+        />
+      </div>
+    )}
+  </SearchProvider>
+);
+```
+
+<a id="searchprovider"></a>
+
+### SearchProvider
+
+`SearchProvider` is what connects Search UI to your components. Components you place
+within `SearchProvider` will be connected to your search experience. It translates the actions users take within
+your search experience into calls to your configured Search API.
 
 Params:
```

Comment:
After this step, things open up. 🔮  

The `SearchProvider` function requires: a `config` object, `parameters`

We have the function and its parameters, introduced a standard (simple) UI, and an example configuration...

I would hesitate to open into "too much" information at this point. Instead, I would choose to move the customization steps down into an isolated context, while developing a "typical/recommended path" and keeping a focused "get started" context.

---

1. Remove the below params table, consider the header closed.
2. This text under Search UI Configuration:

> For the latter, this means that often times you will need to add configuration to Search UI before using a particular component. For example, to use a Facet component to show facet filters, you would first need to provide facets configuration. By configuring facets, you're asking for facet details to be returned on your API calls. These can then be consumed by the Facet component to show facet filters.

I think this is conceptual, and that the relationship between the config object and the component should be introduced earlier, perhaps in the _Creating a Standard(Simple) UI_ section. This information is most digestible when given in the flow of a full example, which that section provides.

3. This text, also under Search UI Configuration...

> Note that this is not the only way to customize API calls. Check out the Customizing API calls - additionalOptions for some more details.
>For this reason, the configuration for Search UI largely follows the same API as the App Search Search API.

It seems to expand into a new context: customization. 

Near the bottom there are two headers: **Using the default styles and layout** & **Using your own styles and layout**. They both fit under "Styling". The customization text quoted above, and all the information below styling with the exception of the Search API response example, should be contained under a customization header and kept in its own conceptual box. 

---

In this way, we build the streamlined, typical path without opening to too many concepts, potentially overwhelming the general audience. The configuration should present: an example, high-level concepts, `SearchProvider` and how `config` and Components relate to it, without pointing us in any other direction.

This is a big piece of structural feedback, so I will cut the review so we can stay on the same page. 📖